sdk/test/akkaserverless-test.js

```diff
@@ -154,6 +156,9 @@ At package.test.json:2:4:
     comp.getComponentType().should.equal('my-type');
     comp.getEntity().getEntityType().should.equal('my-entity-type');
     should.equal(comp.getEntity().getPassivationStrategy(), undefined);
+    expect(comp.getEntity().getForwardHeadersList()).to.have.same.members([
```

Comment:
should we have a quick test for the two possible code-paths?

Reply:
yup, is it supposed to happen at all?

Reply:
Yeah, action should be able to pick up headers as well

Reply:
Covered